packages/cli/src/PublicApi/v1/openapi.yml

```diff
@@ -0,0 +1,307 @@
+---
+openapi: 3.0.0
+info:
+  title: Public n8n API
+  description: n8n Public API
+  termsOfService: https://n8n.io/legal/terms
+  contact:
+    email: hello@n8n.io
+  license:
+    name: Apache 2.0 with Commons Clause
+    url: https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  # Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/RicardoE105/n8n/1.0.0
+  - url: /api/v1
+tags:
+- name: users
+  description: Operations about user
+  externalDocs:
+    description: Find out more about our store
+    url: http://swagger.io
+paths:
+  /users:
+    get:
+      x-eov-operation-id: getUsers
+      x-eov-operation-handler: v1/handlers/Users
+      tags:
+      - users
+      summary: Retrieve all users
+      description: Retrieve all users from your instance. Only available for the instance owner
+      parameters:
+      - name: limit
+        in: query
+        description: The maximum number of items to return
+        required: false
+        schema:
+          type: number
+          example: 100
+          default: 100
+      - name: cursor
+        in: query
+        description: Paginate through users by setting the cursor parameter to a nextCursor attribute returned by a previous request's response. Default value fetches the first "page" of the collection. See pagination for more detail
```

Comment:
Could we link to the pagination info? Will this be provided via the spec file or do you need a docs page for it?

Reply:
I'd vouch for having it as part of the spec file. @StarfallProjects, I assume you'll be including it there ultimately?

Reply:
Pagination info in the spec file is fine.
"I assume you'll be including it there ultimately" - do you mean, will I be including the spec file in the docs? If so yes. If you mean additional pagination info - let's see how clear we can make it in the spec file? And then I'll do extra if needed.